pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -3038,10 +3042,8 @@ public void revertReplaceSegments(String tableNameWithType, String segmentLineag
           // different after updating the lineage entry.
           sendRoutingTableRebuildMessage(tableNameWithType);
 
-          // Invoke the proactive clean-up for segments that we no longer needs in case 'forceRevert' is enabled
-          if (forceRevert) {
```

Comment:
(nit) Can we add one line in the Javadoc to explain that if `forceRevert = true`, `IN_PROGRESS` lineage entry will be reverted.

Reply:
added java doc